modules/adding-node-iso-configs.adoc

```diff
@@ -0,0 +1,153 @@
+// Module included in the following assemblies:
+//
+// *nodes/nodes/nodes-nodes-adding-node-iso.adoc
+
+[id="adding-node-iso-configs_{context}"]
+= Cluster configuration reference
+
+When creating the ISO image, configurations are retrieved from the target cluster and are applied to the new nodes.
+Any configurations for your cluster are applied to the nodes unless you override the configurations in either the `nodes-config.yaml` file or any flags you add to the `oc adm node-image create` command.
+
+[id="adding-node-iso-yaml-config_{context}"]
+== YAML file parameters
+
+Configuration parameters that can be specified in the `nodes-config.yaml` file are described in the following table:
+
+[cols=".^4l,.^3,.^2",options="header"]
+|====
+|Parameter|Description|Values
+
+|hosts:
+|Host configuration.
+|An array of host configuration objects.
+
+|hosts:
+  hostname:
+|Hostname.
+Overrides the hostname obtained from either the Dynamic Host Configuration Protocol (DHCP) or a reverse DNS lookup.
+Each host must have a unique hostname supplied by one of these methods, although configuring a hostname through this parameter is optional.
+|String.
+
+|hosts:
+  interfaces:
+|Provides a table of the name and MAC address mappings for the interfaces on the host.
+If a `NetworkConfig` section is provided in the `nodes-config.yaml` file, this table must be included and the values must match the mappings provided in the `NetworkConfig` section.
+|An array of host configuration objects.
+
+|hosts:
+  interfaces:
+    name:
+|The name of an interface on the host.
+|String.
+
+|hosts:
+  interfaces:
+    macAddress:
+|The MAC address of an interface on the host.
+|A MAC address such as the following example: `00-B0-D0-63-C2-26`.
+
+|hosts:
+  role:
+|Defines whether the host is a `master` or `worker` node.
+Currently, only worker nodes can be added using this process.
+|`worker`.
+
+|hosts:
+  rootDeviceHints:
+|Enables provisioning of the {op-system-first} image to a particular device.
+The node-adding tool examines the devices in the order it discovers them, and compares the discovered values with the hint values.
+It uses the first discovered device that matches the hint value.
+|A dictionary of key-value pairs.
+For more information, see "Root device hints" in the "Setting up the environment for an OpenShift installation" page.
+
+|hosts:
+  rootDeviceHints:
+    deviceName:
+|The name of the device the {op-system} image is provisioned to.
+|String.
+
+|hosts:
+  networkConfig:
+|The host network definition.
+The configuration must match the Host Network Management API defined in the link:https://nmstate.io/[nmstate documentation].
+|A dictionary of host network configuration objects.
```

Comment:
When I originally documented this networkconfig stuff in the [Agent config reference](https://docs.openshift.com/container-platform/4.16/installing/installing_with_agent_based_installer/installation-config-parameters-agent.html#agent-configuration-parameters-optional_installation-config-parameters-agent), I believe @bfournie suggested I only mention this one specific parameter and then just point to the nmstate docs, so that I wouldn't bloat that table with a bunch of already-documented parameters. So that's why there's only this one parameter here.